parser-sdk/nodejs/parser-utils.js

```diff
@@ -0,0 +1,60 @@
+const fs = require("fs");
+const Ajv = require("ajv-draft-04");
+const ajv = new Ajv();
+const addFormats = require("ajv-formats");
+addFormats(ajv);
+const util = require("util");
+var jsonpointer = require("jsonpointer");
+
+// eslint-disable-next-line security/detect-non-literal-fs-filename
+const readFile = util.promisify(fs.readFile);
+const { v4: uuid } = require("uuid");
+
+function addIdsAndDates(findings) {
+  return findings.map((finding) => {
+    return {
+      ...finding,
+      id: uuid(),
+      parsed_at: new Date().toISOString(),
+    };
+  });
+}
+
+// eslint-disable-next-line
+async function validateAgainstJsonSchema(jsonData) {
+  const jsonSchemaString = await readFile(
+    __dirname + "/findings-schema.json",
+    "utf8"
+  );
+  const jsonSchema = JSON.parse(jsonSchemaString);
+  const validator = ajv.compile(jsonSchema);
+  const valid = validator(jsonData);
+  if (!valid) {
+    const errorMessage = generateErrorMessage(validator.errors, jsonData);
+    throw new Error(errorMessage);
+  }
+}
+
+// eslint-disable-next-line
```

Comment:
what eslint rule is this disabling?

Reply:
Same thing for the `validateAgainstJsonSchema` function.
When disabling eslint rules I always try to disable only the specif rule, so other people can understand what is supposed to be disabled. Like with the ` // eslint-disable-next-line security/detect-non-literal-fs-filename` above.

Reply:
They are not needed anymore and I removed them